src/main/resources/anet-schema.yml

```diff
@@ -710,6 +716,55 @@ properties:
           authorizationGroupRelatedObjects:
             "$ref": "#/$defs/labeledField"
 
+      eventSeries:
+        type: object
+        additionalProperties: false
+        required: [status, name, description, hostOrg, adminOrg]
+        properties:
+          status:
+            "$ref": "#/$defs/labeledField"
+          name:
+            "$ref": "#/$defs/inputField"
+          description:
+            "$ref": "#/$defs/inputField"
+          hostOrg:
+            "$ref": "#/$defs/inputField"
+          adminOrg:
+            "$ref": "#/$defs/inputField"
+      event:
```

Comment:
Needs to be a required property

Reply:
All new dictionary fields are now required.